e2e/nomostest/reset.go

```diff
@@ -187,13 +187,8 @@ func ResetRootSyncs(nt *NT, rsList []v1beta1.RootSync) error {
 		defer cancel()
 		go TailReconcilerLogs(ctx, nt, RootReconcilerObjectKey(rsNN.Name))
 
-		// DeletePropagationBackground is required when deleting RSyncs with
```

Comment:
What about using foreground for RepoSync cleanup? Does that come later?

Reply:
RepoSync already tolerated foreground because it didn't use owner references

Reply:
The cleanup code below is still using background by the looks of it

Reply:
weird. fixed.